### PR TITLE
Use errors.New instead of fmt.Errorf

### PR DESCRIPTION
**Pull request by chenyahui**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by liuzengh** [comment]
recheck

**Comment by chenyahui** [comment]
recheck

File: http/transport.go [hunk 1]
```diff
@@ -201,7 +201,7 @@ func (t *ServerTransport) getListener(network, addr string) (net.Listener, error
 		}
 		ln, ok = pln.(net.Listener)
 		if !ok {
-			return nil, fmt.Errorf("invalid net.Listener")
+			return nil, errors.New("invalid net.Listener")
```
Comment (WineChord) [inline http/transport.go:204]: maybe wrapping some information:

```suggestion
			return nil, fmt.Errorf("invalid listener type, want net.Listener, got %T", pln)
```

**Comment by liuzengh** [comment]
> I have read the CLA Document and I hereby sign the CLA

CLA check doesn't seem to be working properly. Could you post a new comment? @chenyahui

**Review by WineChord — APPROVED** [review]
LGTM

**Comment by chenyahui** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**